src/cryptoadvance/specter/templates/wallet/history/components/total_wallet_balances.jinja

```diff
@@ -9,7 +9,7 @@
     <h1>
         <small style="line-height:30px">Total balance:</small><br>
         <span style="color: #fff">
-            {{ wallet.fullbalance | btcunitamount }}
+            <div id="fullbalance_amount">{{ wallet.fullbalance | btcunitamount }}</div>
```

Comment:
```suggestion
            <span id="fullbalance_amount">{{ wallet.fullbalance | btcunitamount }}</span>
```

div causes the BTC to go to new line, should be either a span or include the BTC/ sats in the div.